src/llmcompressor/transformers/finetune/text_generation.py

```
@@ -63,179 +43,3 @@ def train(**kwargs):
     from llmcompressor import train
 
     train(**kwargs)
-
-
-def apply(**kwargs):
```

Please raise a deprecation error?

---

ok